app/exchanges/run_csv/pacbio_sample_sheet_v13.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+# A class spefically for SMRT-Link v13 Sample Sheets, based on the PacbioSampleSheet class
+
+module RunCsv
+  # RunCsv::PacbioSampleSheetV13
+  # Used to generate sample sheets specific to the PacBio pipeline for v13
+  class PacbioSampleSheetV13 < PacbioSampleSheet
+    # Generate a hash of settings for the run
+    def run_settings
+      run = object # Pacbio::Run
+      first_plate = run.plates[0]
```

Comment:
I have been a bit uncomfortable of the fact we are pulling first and second plate out separately rather than looping. Not sure if that is possible to change yet but I think once we are no longer using the old style it might be possible.